src/contracts/index.ts

```diff
@@ -42,14 +44,10 @@ export const getERC721Contract = async (
   return contract;
 };
 
-export const getZAuctionTradeToken = async (
+export const getZnsHub = async (
```

Comment:
The other method in this file is called `get_____Contract` but this one is just `get____`
Can you make this `getZNSHubContract` for consistency please.